api/apps/api/test/access-control/project-acl.fixtures.ts

```diff
@@ -0,0 +1,41 @@
+import * as request from 'supertest';
+import { bootstrapApplication } from '../utils/api-application';
+import { GivenUserIsLoggedIn } from '../steps/given-user-is-logged-in';
+import { GivenProjectExists } from '../steps/given-project';
+import { Repository } from 'typeorm';
+import { getRepositoryToken } from '@nestjs/typeorm';
+import { PublishedProject } from '@marxan-api/modules/published-project/entities/published-project.api.entity';
```

Comment:
Minor, but we may want to create some utility step (or extend the current one) to avoid accessing another unrelated module directly